src/routes/router.tsx

```diff
@@ -79,7 +79,7 @@ export function appRoutes(): RouteObject[] {
 }
 
 const AuthRouter: FunctionComponent<{
-    userManager: (typeof AuthenticationRouter)['userManager'];
+    userManager: Parameters<typeof AuthenticationRouter>[0]['userManager'];
```

Comment:
I see with @TheMaskedTurtle, who find the previous version more readable (I did your version before).

Reply:
We tried it does not work

Reply:
(plus I find this version more understandable)

Reply:
okay